base/intfuncs.jl

```diff
@@ -198,7 +198,7 @@ end
 function ndigits0z(n::Unsigned, b::Int)
     d = 0
     if b < 0
-        d = ndigits0znb(signed(n), b)
+        d = ndigits0znb(n, b)
     else
```

Comment:
I looked in `test/choosetests.jl` to see if there was some of the tests missing in the log.